test/nlp/test_autohf.py

```diff
@@ -128,6 +128,10 @@ def test_hf_data():
     )
 
     automl.predict_proba(X_test)
+
+    X_val = automl._preprocess(X_val)
```

Comment:
Is this intended for user's actual code or just for testing? For user's actual code, we shouldn't call `_preprocess()` as it is an internal function.
A followup question is what would be user's code? Need to add it to the example in documentation.